svc-systems/go.mod

```diff
@@ -3,11 +3,16 @@ module github.com/ODIM-Project/ODIM/svc-systems
 go 1.13
 
 require (
+	github.com/Joker/hpp v1.0.0 // indirect
 	github.com/ODIM-Project/ODIM/lib-dmtf v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201201072448-9772421f1b55
 	github.com/ODIM-Project/ODIM/lib-rest-client v0.0.0-20201201072448-9772421f1b55
 	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201201072448-9772421f1b55
+	github.com/gomodule/redigo v2.0.0+incompatible
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
 	github.com/sirupsen/logrus v1.4.2
 	github.com/stretchr/testify v1.5.1
+	github.com/yudai/pp v2.0.1+incompatible // indirect
```

Comment:
If new packages, please check for license, including the dependent packages.

Reply:
Hi @bharath-b-hpe these came when I did go mod tidy. The only package I introduced is golang reflect package. Moreover github.com/yudai/pp is in the URP plugin. So it is not a new package for ODIM.

Reply:
We dont do OSRB review for your code Arjun. It is for the whole product. So if new licences pop up for URP we need to still ensure we are not importing modules that violate our license expectations.

Reply:
yudai/pp does not have a license. It is forked from k0kubun/pp which has MIT so yudai/pp has MIT license implicitly. But if I remember right we cannot rely on this.

Ashraf have we not done OSRB for URP ?